File: src/main/java/io/zeitwert/ddd/validation/model/enums/CodeValidationLevel.java
package io.zeitwert.ddd.validation.model.enums;

import io.zeitwert.ddd.enums.model.base.EnumeratedBase;
import lombok.Data;
import lombok.experimental.SuperBuilder;

@Data
@SuperBuilder
public class CodeValidationLevel extends EnumeratedBase {

	@Override
	public boolean equals(Object other) {
		return super.equals(other);
	}

	@Override
	public int hashCode() {
		return super.hashCode();
	}

}

File: fm-dddrive/src/main/java/dddrive/ddd/validation/model/enums/CodeValidationLevel.kt
package dddrive.ddd.validation.model.enums

import dddrive.ddd.enums.model.Enumerated

class CodeValidationLevel(
	override val id: String,
	override val defaultName: String,
) : Enumerated {

	override val enumeration: CodeValidationLevelEnum
		get() = CodeValidationLevelEnum.instance

}
